controllers/repository.go

```diff
@@ -84,6 +87,15 @@ func (r *RepositoryController) CreateRepository(c *fireball.Context) (fireball.R
 		return nil, err
 	}
 
+	accounts, err := r.account.Accounts()
```

Comment:
I'm not sure why this is here?

Reply:
We iterate over all the accounts to create a repository policy text every time a new repository is created. In hindsight though, this should probably be in addToRepositoryPolicy.

Reply:
Right, I'm just not sure why this needs to happen during a repository creation though? As far as I can tell, no changes to the `account` table happen during this API call, so this seems like it wouldn't ever change the policy?

Reply:
Not sure I follow. Are you suggesting that I cache a list of accounts so that I don't have to get the list of accounts from the dynamo db table?

Reply:
Oh I see what you mean - the policy is for each repository. So every time you create a new repo, the policy needs to be set for that repository.